core/src/epicli/data/common/ansible/playbooks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+  - hosts: all
```

Comment:
I think this can be removed and maybe you will need to modify in core\src\epicli\data\common\ansible\playbooks\filebeat.yml:
```
- hosts: elasticsearch:kibana # to gather facts
  tasks: []
```